builder/build.go

```diff
@@ -237,3 +240,18 @@ func deDuplicate(buildOptPackages []string) []string {
 	}
 	return retPackages
 }
+
+func getVersion() string {
```

Comment:
What about when there is no git folder available? I think we should check that before running exec. You can find out if you're in a git repo by looking for a `.git` folder.

Reply:
It's addressed. The change is on the next line and will not show as outdated.